Description: fix build with ld --as-needed
 when building with ld --as-needed libraries must be placed
 after object files needing them on the command line so the symbols
 of the libraries are registered as needed.
Author: Julian Taylor <jtaylor.debian@googlemail.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/cdpr/+bug/803174
--- cdpr-2.4.orig/Makefile
+++ cdpr-2.4/Makefile
@@ -16,7 +16,7 @@ CFLAGS = -Wall -W -O2
 # Build for Solaris with the pcap headers/libs one dir up
 #CFLAGS = -DSOLARIS -Wall -I. -I../libpcap-0.7.1 -L../libpcap-0.7.1 -ggdb
 
-LDFLAGS = -lpcap
+LIBS = -lpcap
 
 # Build for Solaris
 #LDFLAGS = -lsocket -lnsl -lpcap
@@ -33,7 +33,7 @@ conffile.o: conffile.c cdp.h cdpr.h
 	gcc -c $(CFLAGS) conffile.c
 
 cdpr: cdpr.o cdprs.o conffile.o
-	gcc $(LDFLAGS) -o cdpr cdpr.o cdprs.o conffile.o
+	gcc $(LDFLAGS) -o cdpr cdpr.o cdprs.o conffile.o $(LIBS)
 
 install: all
 	mkdir -p $(DESTDIR)$(prefix)/sbin/
